src/azure-cli/azure/cli/command_modules/eventhubs/tests/latest/test_cluster_commands.py

```diff
@@ -25,49 +25,48 @@ def test_eh_cluster(self):
             'rg': self.create_random_name(prefix='rg-cluster-', length=20),
             'clustername': self.create_random_name(prefix='eventhubs-clus1-', length=20),
             'namespacename': self.create_random_name(prefix='eventhubs-nscli', length=20),
-            'tags': {'tag1=value1'},
-            'tags2': {'tag2=value2'},
+            'tags': '{tag1:value1}',
+            'tags2': '{tag2:value2}',
```

Comment:
This is also a breaking change.... Is the format of `tag1=value1` still acceptable?

Reply:
Yes both format are acceptable.